fdm-app/app/routes/farm.$b_id_farm.$calendar.gerrit.tsx

```
@@ -157,6 +157,59 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
+    // Strategy: progressively strip trailing content and try closing brackets.
+    const lastCompleteObject = json.lastIndexOf("},")
+    if (lastCompleteObject === -1) return null
+
+    // Take everything up to and including the last complete object in the array
+    const trimmed = json.slice(0, lastCompleteObject + 1)
+
+    // Count unclosed brackets
+    let openBraces = 0
+    let openBrackets = 0
+    let inString = false
+    let escape = false
```

Biome (2.4.13)

[error] Do not shadow the global "escape" property.

(lint/suspicious/noShadowRestrictedNames)